flutter_secure_storage/example/pubspec.yaml

```diff
@@ -24,5 +24,9 @@ dev_dependencies:
     sdk: flutter
   test: any
 
+dependency_overrides:
```

Comment:
As commented, please only use override when developing locally but don't commit this to this repository.

Reply:
Will do. Thank you for the detailed explanation,